docs/node-configuration.md

```diff
@@ -298,6 +298,7 @@ RPC:
   MaxRequestHeaderBytes: 1048576
   MaxWebSocketClients: 64
   MaxWebSocketFeeds: 16
+  MempoolSubscriptionsEnabled: true
```

Comment:
false

They are supposed to be default here (but not all of them, actually, follow this rule).